Clean up

bspears opened the pull request:
Cleaned up spacing from 4 to 2 spaces. Removed duplicate variable declarations. Renamed variables to be more understandable.

EWPS07 committed:
adding my game

EWPS07 committed:
additions to README.md

bspears committed:
Changed spacing to 2 spaces and removed repeating variables.

mel-quark commented on mygame.html:
```diff
@@ -0,0 +1,87 @@
+   
+  userNumber = prompt("Pick another number from 1 to 10 to try to use your NOS!");
+
+  if (userNumber >= computerNumber) {
```
each one of these sections (prompt, if, alert) have the same structure. you could use a function instead, and call it with different strings.

bspears committed:
condensed the game into one function.

the function takes a question, right answer, and wrong answer argument. Now for each question, the function is called with just 3 strings which cut the amount of code in half, and made it much easier to read.
